hopp/simulation/technologies/layout/wind_layout.py

```diff
@@ -73,16 +80,32 @@ class WindBoundaryGridParameters(BaseClass):
     #TODO: rename to border_spacing_ratio?
     border_spacing: float = field(default=0.0)
     #TODO: rename to border_offset_ratio?
-    border_offset: float = field(default=0.0, validator=range_val(0.0, 1.0)) 
-    border_spacing_m: Optional[float] = field(default=None)
-    min_spacing_m: Optional[float] = field(default=None)
+    border_offset: float = field(default = 0.0, validator = range_val(0.0, 1.0)) 
+    border_spacing_m: Optional[float] = field(default = None)
+
+    min_spacing_m: Optional[float] = field(default = 0.0)
+    min_spacing_D: Optional[float] = field(default = 2.0)
+    max_spacing_m: Optional[float] = field(default = 2e6)
+    max_spacing_D: Optional[float] = field(default = 20.0)
 
     grid_angle: float = field(default=0.0, validator=range_val(0.0, 180.0))
     grid_aspect_power: Optional[float] = field(default=None)
     grid_aspect_ratio: Optional[float] = field(default=None)
     row_phase_offset: float = field(default=0.2, validator=range_val(0.0, 1.0))
     
+    min_spacing: float = field(init = False) #min spacing in meters
```

Comment:
If these are still necessary (comment later in the file pertains to this), could add the additional validator above, and set the default value to None (attrs uses its own special None equivalent that is different)?